### DISCUSSION TITLE
Board Request: iCESugar-nano

### DISCUSSION THREAD

**Discussion by djh82uk** [FPGAwars/icestudio#816] [New Boards Requests]
Hi, This could be great board to add, I t could be a great starter/introductory board for many due to it's low cost and simplicity.  Im keen for it to be easier to bring onboard the microcontroller crowd into the FPGA world.

Here are some links for the board:
https://github.com/wuxx/icesugar-nano
https://www.aliexpress.com/item/1005002073816863.html?spm=a2g0s.9042311.0.0.408c4c4dvzc94r

I wish I could help, but I'm no software developer I'm afraid.

**Comment by cavearr** [comment]
Hello! I am reorganizing the issues, trying to give more visibility and be able to better organize and prioritize the work and my roadmap (I will publish it very soon), I am converting some issues that are **new feature requests** as **ideas** on Github **discussions**.

Thanks for understanding.

**Reply by AndriyBatkovich** [reply]
Hi, I would also be happy to have such a great tool on my board!
